README.md

```diff
@@ -44,7 +44,7 @@ distributions. Please ensure that your cluster distribution supports these requi
 - Kubelet certificate needs to be signed by cluster Certificate Authority (or disable certificate validation by passing `--kubelet-insecure-tls` to Metrics Server)
 - Container runtime must implement a [container metrics RPCs] (or have [cAdvisor] support)
 - Network should support following communication:
-  - Control plane to Metrics Server. Control plane node needs to reach Metrics Server's service/pod IP (or node IP if `hostNetwork` is enabled) and ports 443 and 4443. Read more about [control plane to node communication](https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication/#control-plane-to-node). 
+  - Control plane to Metrics Server. Control plane node needs to reach Metrics Server's pod IP and port 10250 (or node IP and custom port if `hostNetwork` is enabled). Read more about [control plane to node communication](https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication/#control-plane-to-node). 
```

Comment:
Why is the service removed here?

Reply:
You can, but behind the scenes either iptables or eBPF will reroute the connection to the pod IP and port.